dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+----
+====
+
+. Verify that the image stream was created:
++
+====
+----
+$ oc get is
+NAME       DOCKER REPO                        TAGS      UPDATED
+my-image   172.30.48.125:5000/test/my-image   latest    3 seconds ago
+----
+====
+
+[[using-image-pull-secrets]]
+== Using Image Pull Secrets
+
+link:../architecture/infrastructure_components/image_registry.html[Docker
+registries] can be secured to prevent unauthorized parties from accessing
+certain images. If you are using OpenShift's integrated Docker registry and are
+pulling from image streams located in the same project, then your pod's service
+account should already have permissions and no additional action should be
+required. If this is not the case, then additional configuration steps are
+required.
+
+[[integrated-registry-authentication-and-authorization]]
+=== Integrated Registry Authentication and Authorization
+
+OpenShift's integrated Docker registry authenticates using the same
+link:../architecture/additional_concepts/authentication.html#api-authentication[tokens]
+as the OpenShift API. To perform a `docker login` against the integrated
+registry, you can choose any user name and email, but the password must be a
+valid OpenShift token.
+In order to pull an image, the authenticated user must have `get` rights on the
+requested `imagestreams/layers`. In order to push an image, the authenticated
+user must have `update` rights on the requested `imagestreams/layers`.
+
+By default, all service accounts in a project have rights to pull any image in
+the same project, and the *builder* service account has rights to push any image
+in the same project.
+
+[[allowing-pods-to-reference-images-across-projects]]
+=== Allowing Pods to Reference Images Across Projects
+
+When using the integrated registry, to allow pods in *project-a* to reference
+images in *project-b*, a service account in *project-a* must be bound to the
+`system:image-puller` role in *project-b*:
+
+----
+$ oc policy add-role-to-user \
+    system:image-puller system:serviceaccount:project-a:default \
+    --namespace=project-b
+----
+
+After adding that role, the pods in *project-a* that reference the default
+service account will be able to pull images from *project-b*.
+
+To allow access for any service account in *project-a*, use the group:
+
+----
+$ oc policy add-role-to-group \
+    system:image-puller system:serviceaccounts:project-a \
+    --namespace=project-b
+----
+
+[[allowing-pods-to-reference-images-from-other-secured-registries]]
+== Allowing Pods to Reference Images from Other Secured Registries
+
+To pull a secured Docker image that is not from OpenShift's integrated registry,
+you must create a *dockercfg* secret and add it to your service account.
+
+If you already have a *_.dockercfg_* file for the secured registry, you can
+create a secret from that file by running:
+
+----
+$ oc secrets new <pull_secret_name> \
```

Comment:
No line break please. And if it's too long, break [line 574](https://github.com/openshift/openshift-docs/pull/1890/files#diff-59e6d0251c8359dd66e65fb35114d276R574) as well.